README.md

```diff
@@ -169,6 +169,19 @@ source .venv/bin/activate && python3 server/app.py
 To check that everything is fine open localhost:8000 in your browser.
 ![](https://i.imgur.com/6SehI5s.png)
 
+## Documentation
+
+Run on Linux:
+```sh
+#Terminal
+pip3 install mkdocs markdown-include mkdocs-material
+mkdocs serve
+```
+
+Then open http://127.0.0.1:8000/ in your browser.
```

Comment:
Please, add dev_addr with 8005 port to mkdocs.yml. sc-machine docs run on 8002 port, scp-machine docs run on 8003 port, sc-component-manager run on 8004 port. See sc-machine mkdocs.yml to learn how to learn how to do it